pure.zsh

```diff
@@ -83,7 +83,7 @@ prompt_pure_precmd() {
 		# make sure working tree is not $HOME
 		[[ "$(command git rev-parse --show-toplevel)" != "$HOME" ]] &&
 		# check check if there is anything to pull
-		command git fetch &>/dev/null &&
+		command git -c gc.auto=0 fetch &>/dev/null &&
```

Comment:
From what I can gather based on [this stackoverflow answer](http://stackoverflow.com/a/5137577) only git fetch should potentially initiate `git-gc`.

Personally I have not run into any reason to disable `gc` though, but I guess it can be useful for big repos, speeding up `git fetch` a bit.

For reference, first try:
```
~/Projects/cdnjs master* ⇣
❯ time git gc
...
git gc  30.35s user 4.42s system 107% cpu 32.261 total
```
second try:
```
❯ time git gc
...
git gc  5.96s user 3.43s system 50% cpu 18.766 total
```